MarkBind/markbind
# Replacing default icon for conversion

**What is the purpose of this pull request?**



**Overview of changes:**
Address the nit stated in #2279 which was brought up in #1866. 
More specifically, I changed the [current file-image icon](https://fontawesome.com/icons/file-image?f=classic&s=solid) to the [house](https://fontawesome.com/icons/house?f=classic&s=solid) icon as suggested in the issue.

**Anything you'd like to highlight/discuss:**

**Testing instructions:**
 Try `markbind init --convert` on a pre-existing set of documents, and ensure that changes are reflected.

**Proposed commit message: (wrap lines at 72 characters)**
Change default favicon for conversions

---

**Checklist:** :ballot_box_with_check:
